### PR TITLE
Weather Component Styling

### LINKED ISSUE #57: Weather Styles
**Is your feature request related to a problem? Please describe.**
The Weather component needs styling

**Describe the solution you'd like**
Style it using styled-components

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Added some styling to the weather component

## Purpose
Enhance UI

## Approach
Styled components

Closes #57

### CODE REVIEW

File: the-toad-tribune/src/layout/Weather.tsx [hunk 1]
```diff
@@ -1,14 +1,27 @@
 import styled from "styled-components";
 import { WeatherProp } from "../api/weatherApi";
+import babysun from "../assets/telebabysun.png"
```
Comment (Gjhnsn) [inline the-toad-tribune/src/layout/Weather.tsx:3]: lol

**Review by connielion — APPROVED** [review]
Looks 👍

**Review by Gjhnsn — APPROVED** [review]
everything looks good!

File: the-toad-tribune/src/layout/Weather.tsx (hunk 1 above)
Comment (kmausolf) [reply the-toad-tribune/src/layout/Weather.tsx:3]: Oh boy.

**Pull request merged**